milvus/datadog_checks/milvus/metrics.py

```diff
@@ -0,0 +1,226 @@
+    'milvus_querynode_sq_req_count': 'querynode.sq.req',
+    'milvus_querynode_sq_req_latency': 'querynode.sq.req.latency',
+    'milvus_querynode_sq_segment_latency': 'querynode.sq.segment_latency',
+    'milvus_querynode_sq_wait_tsafe_latency': 'querynode.sq.wait_tsafe_latency',
+    'milvus_querynode_wait_processing_msg_count': 'querynode.wait_processing_msg',
+    'milvus_querynode_watch_dml_channel_latency': 'querynode.watch_dml_channel_latency',
+    'milvus_rootcoord_collection_num': 'rootcoord.collection_num',
+    'milvus_rootcoord_credential_num': 'rootcoord.credential_num',
+    'milvus_rootcoord_ddl_req_count': 'rootcoord.ddl_req',
+    'milvus_rootcoord_ddl_req_latency': 'rootcoord.ddl_req.latency',
+    'milvus_rootcoord_ddl_req_latency_in_queue': 'rootcoord.ddl_req.latency_in_queue',
+    'milvus_rootcoord_dml_channel_num': 'rootcoord.dml_channel_num',
+    'milvus_rootcoord_entity_num': 'rootcoord.entity_num',
+    'milvus_rootcoord_force_deny_writing_counter': 'rootcoord.force_deny_writing_counter',
+    'milvus_rootcoord_id_alloc_count': 'rootcoord.id_alloc',
+    'milvus_rootcoord_indexed_entity_num': 'rootcoord.indexed_entity_num',
+    'milvus_rootcoord_msgstream_obj_num': 'rootcoord.msgstream_obj_num',
+    'milvus_rootcoord_num_of_roles': 'rootcoord.num_of_roles',
+    'milvus_rootcoord_partition_num': 'rootcoord.partition_num',
+    'milvus_rootcoord_produce_tt_lag_ms': 'rootcoord.produce_tt_lag_ms',
+    'milvus_rootcoord_proxy_num': 'rootcoord.proxy_num',
+    'milvus_rootcoord_qn_mem_high_water_level': 'rootcoord.qn_mem_high_water_level',
+    'milvus_rootcoord_sync_timetick_latency': 'rootcoord.sync_timetick_latency',
+    'milvus_rootcoord_timestamp': 'rootcoord.timestamp',
+    'milvus_rootcoord_timestamp_saved': 'rootcoord.timestamp_saved',
+    'milvus_runtime_info': 'runtime_info',
+    'milvus_storage_kv_size': 'storage.kv_size',
+    'milvus_storage_op_count': 'storage.op',
+    'milvus_storage_request_latency': 'storage.request_latency',
+    'bf_search_cnt': 'bf_search_cnt',
+    'bitset_ratio': 'bitset_ratio',
+    'build_latency': 'build_latency',
+    'cache_hit_cnt': 'cache_hit_cnt',
+    'diskann_range_search_iters': 'diskann.range_search_iters',
+    'diskann_search_hops': 'diskann.search_hops',
+    'diskann_bitset_ratio': 'diskann_bitset_ratio',
+    'exec_latency': 'exec_latency',
+    'filter_connectivity_ratio': 'filter.connectivity_ratio',
+    'filter_mv_activated_fields_cnt': 'filter.mv.activated_fields_cnt',
+    'filter_mv_change_base_cnt': 'filter.mv.change_base_cnt',
+    'filter_mv_only_cnt': 'filter.mv.only_cnt',
+    'filter_mv_supplement_ep_bool_cnt': 'filter.mv.supplement_ep_bool_cnt',
+    'go_gc_duration_seconds': 'go.gc_duration_seconds',
+    'go_goroutines': 'go.goroutines',
+    'go_info': 'go.info',
+    'go_memstats_alloc_bytes': 'go.memstats.alloc_bytes',
+    # 'go_memstats_alloc_bytes_total': 'go.memstats.alloc_bytes_total',
```

Comment:
```suggestion
    'go_memstats_alloc_bytes': {'name': 'go.memstats.alloc_bytes', 'type': 'native_dynamic'},
```

Type native collects them as a both gauge and then the _total version as a count